### ISSUE TITLE
Race condition on click2vox

### ISSUE THREAD

**Issue by rviscarra** [voxbone-workshop/click2vox-com#733]
Sometimes a call would fail to start because the pages tries to start the call before the library has initialized itself. 

First I saw it on my custom page for the Alexa 911 call, but I also saw it on voxconf.me.

![Console Trace from voxconf.me](https://user-images.githubusercontent.com/363249/30451276-57eabeb8-9950-11e7-870f-fabf15dcee08.png)

This is the simplest fragment of code to reproduce it:
``` HTML
  <script>
    var did = window.location.pathname.split('/').pop();

    var click2voxObject = {
      did: did,
      server_url: "https://click2vox.com",
      use_default_button_css: "true",
      div_css_class_name: "btn-style-round-a",
      placement: "bottom-right",
      draggable: "true",
      https_popup: "false",
      caller_id: "emergencyCall"
    };
  </script>
  <script src="https://www.voxbone.com/click2vox/click2vox.js"></script>
  <script>
    document.addEventListener('DOMContentLoaded', function() {
      function startCall() {
        var voxboneLoaded = window.voxbone !== undefined;
        if(voxboneLoaded) {
          click2voxObject.makeCall(true);
        } else {
          setTimeout(startCall, 1000);
        }
      }
      startCall();
    });
  </script>
```
I suspect it is a race condition, so the error doesn't always happen. I'm able to trigger it ~ 40% of the time with the code fragment from above.

**Comment by san983** [comment]
Two things to check here:
- [ ] Can we easily fix this in a v2.2.x release? 
- [ ] Does this happen also with v3.x? (I'm really interested to prevent this here)

**Comment by agonza1** [comment]
Seems that this fixes it for both: #734
